src/Symfony/Component/Routing/RequestContext.php

```diff
@@ -269,6 +269,20 @@ public function setParameters(array $parameters)
         return $this;
     }
 
+    /**
+     * Adds some parameters.
+     *
+     * @param array $parameters The parameters
+     *
+     * @return $this
+     */
+    public function addParameters(array $parameters)
```

Comment:
Better use `setParameters` instead of adding a method.

Reply:
Done